### ISSUE TITLE
Suggestion // Add iodéOS in section Operating Systems - Android

### ISSUE THREAD

**Issue by AlphaElwedritsch** [pluja/awesome-privacy#302]
Hi all,
is it possible to add the privacy oriented custom ROM iodéOS in the section Operating Systems - Android?
this plays in the same league as the other custom ROMs mentioned there

https://iode.tech

Thank you very much

**Comment by monokrome** [comment]
This is essentially a fork of LineageOS, but it doesn't seem to be open source. I don't think that a closed source android OS can be safely considered "privacy respecting" - especially when its based on an already privacy respecting open source alternative. Is there source code for this OS that I've missed somewhere?

**Comment by AlphaElwedritsch** [comment]
yes, you missed
it is not just a fork of LOS. It is a independent privacy and security based ctusom ROM with LOS base. Same as DivestOS 

it is opensource completetly since around half a year now:
https://gitlab.com/iode

**Comment by 0xDE57** [comment]
Closed in 5bc8aa4

**Comment by monokrome** [comment]
@AlphaElwedritsch Those repositories don't have any actual code in them, they're just a bunch of metadata files. How does it prove that the distro is privacy respecting?

Also, the link you gave goes to a page that doesn't even exist: https://iode.tech/en/news/iodeos-is-open-source/

CC @0xDE57 @pluja - did y'all confirm any of this?...

EDIT: Also, yes, it is definitely a fork of LineageOS..

**Comment by 0xDE57** [comment]
@monokrome 
Hmm. Digging through the gitlab, it does seem to be mostly config files and very little actual code... Some of the device trees are literally just a few binaries...

It also appears that most of these files have not been updated since: Jun 25, 2024. So they are probably also lagging on security patches.

---

Personally I wouldn't use iode or /e/. I would instead use GrapheneOS, or LineageOS if you don't have a pixel. LineageOS at least gets regular updates (if your device is not too old) and a much larger community and better support. I have used de-googled LineageOS for many years and it has always been good to me. 

Anecdotes aside, I closed my PR because I couldn't really vouch for the project. And also some concerns about /e/ and iodéOS being based in France: https://github.com/pluja/awesome-privacy/pull/543#issuecomment-4018061819

¯\\_(ツ)_/¯ 

 It's pluja's call to make.

**Comment by monokrome** [comment]
@0xDE57 Yeah, but even after you agree its hard to vouch for, it was still added to the list. Very confusing.

https://github.com/pluja/awesome-privacy/commit/5bc8aa42013fb22bb1c893588c63b4c45c93f1f9#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1019

EDIT: Wait, what's wrong with France?

**Comment by 0xDE57** [comment]
@monokrome 

I did not add it.

**Comment by monokrome** [comment]
@0xDE57 I know, I wasn't claiming you did. Apologies if it seemed that way.

**Comment by 0xDE57** [comment]
> Wait, what's wrong with France?

Perhaps read the linked links.
